src/main/java/com/blamejared/compat/thermalexpansion/Centrifuge.java

```diff
@@ -64,6 +84,59 @@ protected String getRecipeInfo() {
             return LogHelper.getStackDescription(input);
         }
     }
+
+    private static class AddMob extends BaseAction {
+        private ItemStack[] outputs;
+        private Integer[] chances;
+        private String entityId;
+        private Integer energy;
+        private FluidStack fluid;
+        private Integer xp;
+
+        public AddMob(ItemStack[] outputs, Integer[] chances, String entityId, Integer energy, FluidStack fluid, Integer xp) {
+            super("Centrifuge");
+            this.outputs = outputs;
+            this.chances = chances;
+            this.entityId = entityId;
+            this.energy = energy;
+            this.fluid = fluid;
+            this.xp = xp;
+        }
+
+        @Override
+        public void apply() {
+            if (!ItemMorb.validMobs.contains(entityId)) {
+                CraftTweakerAPI.logError("Not a valid morb entity: " + entityId);
+                return;
+            }
+
+            if (energy==null) { this.energy=CentrifugeManager.DEFAULT_ENERGY*2; }
```

Comment:
Will always be false since the only time you create assign to this energy field is during construction.
The only place where you construct these objects is in `addRecipeMob`.
`addRecipeMob` accepts primitives (ints) for the parameters, which will never be null.

Reply:
Excellent point, I've resolved this.